.clang-format

```diff
@@ -0,0 +1,215 @@
+---
```

Comment:
Added this file for consistency with other contributors. Happy to remove it if it's not necessary.